vdb-bench-assembly/plugins/vdb-bench-dataservice/dsPageService.js

```diff
@@ -14,9 +14,9 @@
         .module('vdb-bench.dataservice')
         .factory('DSPageService', DSPageService);
 
-    DSPageService.$inject = ['SYNTAX', 'CONFIG'];
+    DSPageService.$inject = ['SYNTAX', 'CONFIG', '$translate'];
 
-    function DSPageService(syntax, config) {
+    function DSPageService(syntax, config, translate) {
```

Comment:
While certainly fine as it is, it seems to be preferred practice that if an injection name starts with a $ then its related var should retain that in the function. '$' vars denote built-in angular services and have a special status as a result. The '$' just ensures this status remains obvious in the implementation.